tests/core/coverage/coverage_test.go

```diff
@@ -173,15 +173,19 @@ func TestPanic(t *testing.T) {
 
 func TestCoverage(t *testing.T) {
 	t.Run("without-race", func(t *testing.T) {
-		testCoverage(t, "set")
+		testCoverage(t, "set", "bazel-testlogs/a_test/coverage.dat")
 	})
 
 	t.Run("with-race", func(t *testing.T) {
-		testCoverage(t, "atomic", "--@io_bazel_rules_go//go/config:race")
+		testCoverage(t, "atomic", "bazel-testlogs/a_test/coverage.dat", "--@io_bazel_rules_go//go/config:race")
+	})
+
+	t.Run("with-combined-lcov", func(t *testing.T) {
+		testCoverage(t, "set", "bazel-out/_coverage/_coverage_report.dat", "--combined_report=lcov")
```

Comment:
The `testCoverage` function below disables lcov in favor of the Go cover format, so this test can't pass.

Maybe move this info `lcov_coverage_tests.go`?

Reply:
`lcov_coverage_tests.go` already has this before https://github.com/bazelbuild/rules_go/pull/3294.